core/src/main/java/discord4j/core/object/entity/Webhook.java

```diff
@@ -223,6 +284,75 @@ public String getToken() {
                 .map(data -> new Webhook(gateway, data));
     }
 
+    /**
+     * Requests to edit this webhook.
+     * Does not require the MANAGE_WEBHOOKS permission.
+     * The channel and reason are ignored.
```

Comment:
Would it be a good idea to use a different spec without those fields here?

Reply:
I decided to do that ^-^